container.yaml

```diff
@@ -0,0 +1,11 @@
+# new for 4.5
+# https://osbs.readthedocs.io/en/latest/users.html#replacing-pullspecs
```

Comment:
I don't think we need this file in upstream, I can add it in the midstream repo automatically since it's red hat specific

Reply:
I was shooting blindly at the end, copying files over from the web-terminal-operator repo :D